src/main/java/com/treasuredata/client/AbstractTDClientBuilder.java

```diff
@@ -277,6 +280,12 @@ public BuilderImpl setProxy(ProxyConfig proxyConfig)
         return self();
     }
 
+    public BuilderImpl setRetryStrategy(String retryStrategy)
```

Comment:
Why not passing BackoffStrategy enum here? Then, we can ensure the user will always use one of the valid BackoffStrategy enum items.